src/uk/gov/hmcts/contino/AppPipelineDsl.groovy

```diff
@@ -43,8 +43,9 @@ class AppPipelineDsl extends CommonPipelineDsl implements Serializable {
     config.parallelCrossBrowsers = browsers
   }
 
-  void enableSecurityScan(String urlExclusions = "", int timeout = 120) {
+  void enableSecurityScan(String urlExclusions = "", isFrontend = false, int timeout = 120) {
```

Comment:
instead of using positional arguments you could use a map:

```suggestion
  void enableSecurityScan(Map<String, String> args = [:]) {
        def config = [
            urlExclusions: "",
            isFrontend: false,
            timeout: 120
    ] << args
  }
```